joblib/testing.py

```diff
@@ -46,10 +47,12 @@ def assert_raises_regex(expected_exception, expected_regexp,
 try:
     SkipTest = unittest.case.SkipTest
 except AttributeError:
-    # Python <= 2.6, we still need nose here
-    SkipTest = nose.SkipTest
+    SkipTest = _pytest.runner.Skipped
 
+skipif = pytest.mark.skipif
 with_setup = nose.tools.with_setup
+fixture = pytest.fixture
+parametrize = pytest.mark.parametrize
```

Comment:
Oh yes, Probably I'll use it in immediately next commit after this PR. For now I'll remove this line. I got in the flow and wrote both fixture and parametrize at once !